projects/edk2/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://www.tianocore.org"
+language: c
+fuzzing_engines:
+   - libfuzzer
+sanitizers:
+   - address
+   - undefined
+primary_contact: "edk2-hbfa-ossfuzz@intel.com"
```

Comment:
You will probably have issues accessing the bugs for this project. I recommend putting actual individual's email addresses